Description: Fix typo in manpage
Author: Ryan Kavanagh <rak@debian.org>
Forwarded: https://github.com/OpenSMTPD/OpenSMTPD-extras/pull/28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: opensmtpd-extras/extras/wip/filters/filter-pause/filter-pause.8
===================================================================
--- opensmtpd-extras.orig/extras/wip/filters/filter-pause/filter-pause.8	2016-02-20 09:18:52.579410061 -0500
+++ opensmtpd-extras/extras/wip/filters/filter-pause/filter-pause.8	2016-02-20 17:49:44.693448362 -0500
@@ -39,7 +39,7 @@
 Set the number of
 .Ar seconds
 to pause before sending the initial greeting.
-The accepted values are withing the range of 1s and 300s.
+The accepted values are within the range of 1s and 300s.
 .Pp
 The default
 .Ar seconds
